frontend/src/pages/participatedMentoring/ReviewButton/ReviewButton.tsx

```diff
@@ -13,7 +13,9 @@ interface ReviewButtonProps {
   isReviewed: boolean;
   status: StatusType;
   onReviewButtonClick: () => void;
```

Comment:
p4: 만약 전달하는 함수를 분리했다면 여기 인터페이스도 이렇게 변할 수 있을 것 같아!

```suggestion
onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
```